backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ConnectionDefinitionImpl.java

```diff
@@ -312,12 +311,11 @@ public EList<Type> getTargetType() {
      */
     @Override
     public EList<Usage> getConnectionEnd() {
-        List<Usage> ends = this.getOwnedRelationship().stream()
-                .filter(FeatureMembership.class::isInstance)
-                .map(FeatureMembership.class::cast)
-                .flatMap(fm -> fm.getOwnedRelatedElement().stream())
+        List<Usage> ends = this.getAssociationEnd().stream()
```

Comment:
SysML specification 8.3.13.3: connectionEnd redefines associationEnd and the end `Features` of a `ConnectionDefinition` must be `Usages`